magnet/excitation-data/it-lens.txt

```diff
@@ -0,0 +1,69 @@
+# HEADER
+# ======
+# label             it-lens
+# harmonics         1
+# main_harmonic     1 normal
+# rescaling_factor  1.0
+# units             Ampere  T T
+
+# EXCITATION DATA
+# ===============
+-0125.00  -2.000000e+01 +0.000000e+00
```

Comment:
@anacso17 , it does not affect the actual use of the PS, but I think it is more appropriate to limit the excdata to +/- 10A, as they are used in FBP-model power supplies.

Reply:
done!